### DISCUSSION TITLE
[1967430] 幽灵诡计【Ghost Trick: Phantom Detective】

### DISCUSSION THREAD

**Discussion by miaor123** [pjy612/SteamManifestCache#475] [Share]
`appid:1967430`
keys:
```
1967431;ef7589c5444c9766ccd3e2cb0e66dafdca7b6eec90068702be49b085059b790a
```

[1967430.zip](https://github.com/pjy612/SteamManifestCache/files/14209182/1967430.zip)
SAI找不到了因此没有CODE。。
邮箱：[2814589496@qq.com](mailto:2814589496@qq.com)

**Comment by pjy612** [comment]
* 麻烦 按照格式 #26

**Reply by miaor123** [reply]
已修改
